docs/docs/explore/project-chat.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "AI Chat in Rill Cloud"
+description: Chat with your data using natural language in Rill Cloud
+sidebar_label: "AI Chat"
+sidebar_position: 05
+---
+
+## Overview
+
+AI Chat in Rill Cloud allows you to have natural language conversations with your data directly in your browser. Instead of building queries or navigating through dashboards, simply ask questions in plain English and get instant insights backed by your metrics views—complete with **interactive visualizations** and **direct links** to your existing dashboards for deeper exploration.
+
+AI Chat is powered by [Rill's Model Context Protocol (MCP)](/explore/mcp) integration, which ensures that responses are accurate, governed, and consistent with the metrics displayed in your dashboards. By querying data with **predefined measures and dimensions**, you can trust that the answers you receive are as reliable as the data in your Rill dashboards. Plus, responses include canvas dashboard components that visualize your data and automatically link back to the relevant dashboards with filters pre-applied.
```

Comment:
I do really like this spirit of progressive disclosure/complexity, where we first introduce MCP then second introduce AI Chat. But, for the current user flow, given the user first enters the AI chat, then second has the option to "Connect your own client", I think it might be better to put the MCP docs page second.